packages/core/src/index.ts

```diff
@@ -1,6 +1,8 @@
 /**
  * @module @aerogearservices/core
  */
+import console from "loglevel";
+console.setDefaultLevel(console.levels.WARN);
```

Comment:
Maybe INFO?

Reply:
-1. This is default for users and info should not appear by default in the browser, cordova etc. We can change it.